recipes/TXM_Sandbox/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "TXM_Sandbox" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/fa/57/4d28f168e88f19781c615a8b7a0566dffb1365550a3570c7289d8bbc0a07/TXM_Sandbox-0.1.tar.gz 
+  sha256: d9fd3cc1fa6b94235cfdb77aa66d3dc3f9d868fb2999a754ed62e47d01e2f2fb 
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Comment:
Minor comment, you can use `{{ PYTHON }} -m pip install . -vv`, no need to specify `--no-deps` manually when building on conda-forge

Reply:
changed as suggested.